app/models/database.rb

```diff
@@ -240,6 +251,22 @@ def self.libguides_export
     end
   end
 
+  # Generates a CSV to export link tracking data.
+  # @author Tracy A. McCormick
+  # @return csv object
+  def self.linktracking_export
+    headers = %w[
+      name count
+    ]
+    CSV.generate(headers: true) do |csv|
+      csv << headers
+      # get everything else
+      all.find_each do |database|
```

Comment:
This looks wrong. 
all should follow an active record object.  Otherwise you are grabbing everything thing from the database. Also you can just use an each statement if you do it properly instead of a find_each.

Reply:
This was discussed. No change made.